packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -44,39 +45,19 @@ function registerDebugHandlers(): vscode.Disposable {
     async event => {
       if (event && event.session && event.session.type === DEBUGGER_TYPE) {
         if (event.event === GET_LINE_BREAKPOINT_INFO_EVENT) {
-          const sfdxApex = vscode.extensions.getExtension(
-            'salesforce.salesforcedx-vscode-apex'
+          console.log('in registerDebugHandlers, getting line breakpoint info');
```

Comment:
@JimSuplizio - Can we remove this `console.log`?

CC: @DatGuyJonathan , @lcampos